src/plugins/discover/public/application/components/doc_viewer_links/doc_viewer_links.scss

```diff
@@ -0,0 +1,21 @@
+.osdDocViewerLinks {
```

Comment:
I guess we don't lint scss files?

I see most of our scss files are two space indented and I think some of the indention could use some loving.

Reply:
done